### PR TITLE
Flag/Quest ID Checker

### PR DESCRIPTION
**Pull request by JevLOMCN**
New Search Box added to PlayerInfoForm, This will allow server owners to search if a player has a Flag set as active or inactive.

Also added Map Title (Name) to display with Current Location

### CODE REVIEW

File: Server.MirForms/Account/PlayerInfoForm.cs [hunk 1]
```diff
@@ -133,5 +133,122 @@ private void OpenAccountButton_Click(object sender, EventArgs e)
...
+                if (questID < 1)
+                {
+                    QuestResultLabel.Text = "Invalid Quest ID";
+                    QuestResultLabel.ForeColor = Color.Red;
+                    return;
+                }
+
+                bool isQuestComplete = Character.CompletedQuests.Contains(questID);
+
+                QuestResultLabel.Text = $"Quest {questID} is {(isQuestComplete ? "Complete" : "Not complete")}";
```
Comment (Suprcode) [inline Server.MirForms/Account/PlayerInfoForm.cs:218]: maybe needs to be abit more sophisticated for quest lookup. As well as completed also check active quests.

File: Server.MirForms/Account/PlayerInfoForm.cs [hunk 2]
```diff
@@ -133,5 +133,122 @@ private void OpenAccountButton_Click(object sender, EventArgs e)
...
+
+            if (int.TryParse(flagTextBox.Text, out int flagIndex))
+            {
+                if (flagIndex >= 0 && flagIndex < Character.Flags.Length)
+                {
+                    bool flagValue = Character.Flags[flagIndex];
+
+                    if (flagValue)
+                    {
+                        ResultLabel.Text = $"Flag {flagIndex} ACTIVE.";
```
Comment (Suprcode) [inline Server.MirForms/Account/PlayerInfoForm.cs:154]: change "ACTIVE." to "is active"
change "INACTIVE." to "is inactive"

so it matches the output format of quests

**Review by Suprcode — CHANGES_REQUESTED** [review]

File: Server.MirForms/Account/PlayerInfoForm.cs (hunk 2 above)
Comment (JevLOMCN) [reply Server.MirForms/Account/PlayerInfoForm.cs:154]: This has been adjusted now

File: Server.MirForms/Account/PlayerInfoForm.cs (hunk 1 above)
Comment (JevLOMCN) [reply Server.MirForms/Account/PlayerInfoForm.cs:218]: Not sure how to change the code, I know that 0 is not complete and 1 is complete.

**Comment by JevLOMCN** [comment]
All adjustments have been made.

**Pull request merged**